CHANGES.md

```diff
@@ -1,5 +1,24 @@
 # Changes
 
+## edge-23.3.4
+
+This edge release further enhances the OutboundPolicies API used by the proxy to
+route outbound traffic, and continues extending the HTTPRoute resource's Status
+field. It also starts integrating circuit-breaking functionality into the proxy,
+which will be configurable in a subsequent iteration.
+
+* Continued iterating over the HTTPRoute's Status field, by extending support
```

Comment:
take it or leave it: "iterating over over" makes it sound like we have a loop over a list of HTTPRoute status fields, or something. maybe "Continued iterating on" instead?

Reply:
Sounds good, I'll update that. I always have trouble with prepositions :man_shrugging: